components/base/kinematicbase/ptgKinematics.go

```diff
@@ -0,0 +1,122 @@
+// Package kinematicbase contains wrappers that augment bases with information needed for higher level
+// control over the base
+package kinematicbase
+
+import (
+	"context"
+	"errors"
+	"math"
+	"time"
+
+	"github.com/golang/geo/r3"
+	"go.uber.org/multierr"
+	utils "go.viam.com/utils"
+
+	"go.viam.com/rdk/components/base"
+	"go.viam.com/rdk/motionplan/tpspace"
+	"go.viam.com/rdk/referenceframe"
+	rdkutils "go.viam.com/rdk/utils"
+)
+
+// Define a default speed to target for the base in the case where one is not provided.
+const defaultBaseMps = 0.6
+
+const (
+	ptgIndex int = iota
+	trajectoryIndexWithinPTG
```

Comment:
what is the value of the constants?

Reply:
https://www.gopherguides.com/articles/how-to-use-iota-in-golang

Reply:
ooooo, this is cool!